scripts/lib.star

```diff
@@ -441,6 +443,38 @@ def build_frontend_step(edition, is_downstream=False):
         ],
     }
 
+def generate_packages_docs(edition, lint=False):
+    if edition == 'enterprise':
+        return None
+
+    if lint:
+        return {
+            'name': 'generate-packages-docs',
+            'image': build_image,
+            'depends_on': [
+                'build-frontend'
+            ],
+            'environment': {
+                'GRAFANA_MISC_STATS_API_KEY': {
+                    'from_secret': 'grafana_misc_stats_api_key',
+                },
```

Comment:
This is a problem, since this secret won't be shared with PRs, for security reasons.

Reply:
aha, so lets remove this one for now. I think it is enough just to fail the build.

Reply:
Yeah, just testing should be enough for PRs, as you say.